js/ui/file_uploader.js

```diff
@@ -1597,6 +1363,237 @@ FileUploader.__internals = {
 };
 ///#ENDDEBUG
 
+var FileUploadStrategyBase = Class.inherit({
+    ctor: function(fileUploader) {
+        this.fileUploader = fileUploader;
+    },
+    upload: function(file) {
```

Comment:
Add an empty line after the methods. For example, please see the button.js file.